### ISSUE TITLE
Feature: Add fork button

### ISSUE THREAD

**Issue by yg** [open-sauced/open-sauced#743]
### Is your feature request related to a problem? Please describe.

Add fork button near to triage button

### Describe the solution you'd like

We can use this query to detect fork https://serve.onegraph.com/short/CYT92

### Additional context

![Screenshot](https://user-images.githubusercontent.com/61746440/85859645-1e29d400-b7db-11ea-80b4-44a6b0a9923a.png)

**Comment by JapneetSingh5** [comment]
This would be a nice addition!

**Comment by bdougie** [comment]
I love this idea :shipit:

**Comment by sgrove** [comment]
@bdougie Heads up, here's a mutation to fork a repo: https://serve.onegraph.com/short/KKQB3F

**Comment by bdougie** [comment]
For anyone that would like to take this, you can add the following to [apiGraphQL.js
](https://github.com/open-sauced/open-sauced/blob/master/src/lib/apiGraphQL.js)
```
mutation ForkRepository(
  $repoName: String!
  $repoOwner: String!
) {
  gitHub {
    createFork_oneGraph(
      input: { repoName: $repoName, repoOwner: $repoOwner }
    ) {
      clientMutationId
      repository {
        id
        url
      }
    }
  }
}
```

**Comment by MKrupauskas** [comment]
I'd like to take this issue on! What should happen after the user clicks the button and the fork mutation is executed?

**Comment by yg** [comment]
Hey @MKrupauskas! Thanks for jumping in :purple_heart: 

> What should happen after the user clicks the button and the fork mutation is executed?

1. Before forking the button, it displays with the text "Fork {count}"
2. On clicking the Fork button the button should be disabled until success (To prevent multiple forks)
3. After successful execution of the mutation, the button displays "Go to fork"

WDYT @bdougie @MKrupauskas?

You can self assign this issue by replying with `.take` in this issue!

Please feel free to join our Discord https://discord.gg/6trQuJw :pizza:

**Comment by bdougie** [comment]
> After successful execution of the mutation, the button displays "Go to fork"

Instead, it should state, "View Fork"

**Comment by MKrupauskas** [comment]
.take

**Comment by bdougie** [comment]
Fixed in #719
